packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -0,0 +1,125 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import React from "react";
+import { Body1, Body1Strong, Subtitle1, makeStyles } from "@fluentui/react-components";
+import { DynamicComposedChart } from "./graphs";
+
+const useStyles = makeStyles({
+	flexColumn: {
+		display: "flex",
+		flexDirection: "column",
+	},
+	flexRow: {
+		display: "flex",
+		flexDirection: "row",
+	},
+	/**
+	 * Where the graph is described
+	 */
+	graphAboutContainer: {
+		marginTop: "15px",
+	},
+	/**
+	 * Where the op lifecycle phases are described
+	 */
+	opPhasesSection: {
+		marginTop: "15px",
+	},
+});
+
+/**
+ * Page that shows op latency metrics.
+ * @remarks TODO: Once Op Latency data is available from {@link messageRelay}, the op latency data should be passed into the graph instead of this test data set.
+ */
+export function OpLatencyView(): React.ReactElement {
+	const styles = useStyles();
+
+	return (
+		<div style={{ width: "100%", height: "300px" }} data-testid="test-op-latency-view">
```

Comment:
Last bit of styles to extract.

Reply:
Done. Also removed the last telemetry reference.